src/Botty.ts

```diff
@@ -97,6 +100,10 @@ export default class Botty {
         });
         console.log("Initialised listeners.");
     }
+    
+     public onVersion(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
+        message.channel.send(this.version);
```

Comment:
Maybe something like `"Current Version: " + version` instead of just dumping the version in chat?